frontend/src/APIClients/queries/TestSessionQueries.ts

```diff
@@ -4,6 +4,9 @@ const GET_TEST_SESSION_BY_ACCESS_CODE = gql`
   query TestSessionByAccessCode($accessCode: String!) {
     testSessionByAccessCode(accessCode: $accessCode) {
       id
+      test {
+        id
+      }
```

Comment:
so i realized we actually store the entire test object inside a test session. i don't think we need to do that since we should probably only query for the test in the `AssessmentSummary` page (i.e after student authentication is complete). given this, should i make a PR to only store the `test_id` in a `test_session` object? or is there another purpose for this

Reply:
hmm yea, make sense to have a separate query (can't think of a use case off the top of my head). we should probably re-evaluate our other functions as well (i.e. if we need the ResponseDTOs or just the id)

Reply:
makes sense. added a [ticket](https://www.notion.so/uwblueprintexecs/Refactor-Test-Session-model-to-omit-Test-object-91e7fabcbc464fec8b6cbaefac910353?pvs=4) for storing only the id! i'll work on it after we finish the backend changes for the teacher and student dashboards, in case we do end up needing it.

for the other functions, agree. maybe we can create tickets as we complete the backend work coming up and then at the end, we can do a full cleanup?

Reply:
sg!